src/index.tsx

```diff
@@ -21,25 +21,17 @@ interface WorkerLoad<Data> {
   /** The worker's returned value stored in a state */
   data: Data;
 
-  /** An optional object that contains any thrown errors, if any, and a retry function */
-  error?: RetryWorkerError;
+  /** The error thrown by the worker's call */
+  error?: Error;
```

Comment:
At the line `13` you're referencing `RetryWorkerError`, should we remove it? Besides that, what do you think about moving it to closer to the `useWorkerLoad`?

Reply:
Good catch. I think it'd be best to keep this declaration where it is because it's the only interface type being exported by the module, all the other types are related to the functions.

Reply:
Actually, I haven't payed close attention and `ValuablePromise` is not declared at the top of the file, I'll move it up.